packages/hop-node/src/watchers/classes/Bridge.ts

```diff
@@ -66,10 +68,15 @@ export default class Bridge extends ContractBase {
     }
     this.db = getDbSet(this.tokenSymbol)
     const bridgeDeployedBlockNumber = config.tokens[this.tokenSymbol]?.[this.chainSlug]?.bridgeDeployedBlockNumber
+    const l1CanonicalTokenAddress = config.tokens[this.tokenSymbol]?.[Chain.Ethereum]?.l1CanonicalToken
     if (!bridgeDeployedBlockNumber) {
       throw new Error('bridge deployed block number is required')
     }
-    this.bridgeDeployedBlockNumber = config.tokens[this.tokenSymbol]?.[this.chainSlug]?.bridgeDeployedBlockNumber
+    if (!l1CanonicalTokenAddress) {
+      throw new Error('l1 token address is required')
```

Comment:
suggestion: capitalize `L1` for readability

Reply:
Resolved with 46c229881e4ee54f3f6259e30a89cbed5fbead07